src/Sylius/Behat/Page/Admin/Crud/UpdatePage.php

```diff
@@ -65,6 +65,20 @@ public function checkValidationMessageFor($element, $message)
     /**
      * {@inheritdoc}
      */
+    public function hasResourceValues(array $parameters)
```

Comment:
Same function I have seen at [this PR](https://github.com/Sylius/Sylius/pull/4742/files#diff-efd046c305a68f775819d3f42bacf350R45) It would be worthy to add it once Maybe separate PR?